wicketd/tests/integration_tests/commands.rs

```diff
@@ -33,7 +33,13 @@ fn test_wicketd_openapi() {
         .expect("stdout was not valid OpenAPI");
 
     // Check for lint errors.
-    let errors = openapi_lint::validate(&spec);
+    let mut errors = openapi_lint::validate(&spec);
+    // XXX: For types like `StepEvent<WicketdEngineSpec>`, schemars generates
+    // names like `StepEvent_for_WicketdEngineSpec`. Filter them out since we
+    // can't control those names.
+    errors.retain(|s| {
+        !(s.contains("that is not PascalCase") && s.contains("_for_"))
+    });
```

Comment:
This is no longer required as of #2863.

Reply:
I'm pointed at #2697 which I think (?) doesn't have that yet.

Reply:
Yeah, #2697 doesn't have that yet. Feel free to combine this with #2697 and rebase :)

Reply:
Rebased and removed this in 3fc0ac021d742a811dfc7feb56beec211e109a26